app/util/legUtils.js

```diff
@@ -3,6 +3,26 @@ import get from 'lodash/get';
 import { BIKEAVL_UNKNOWN } from './vehicleRentalUtils';
 import { getRouteMode } from './modeUtils';
 
+function getNestedValue(obj, path) {
```

Comment:
Since this is in legUtils, I would recommend to add a small documentation what these functions are and how they can be used.

Reply:
getNestedValue could also reside in a more common util file if such exists.